### DISCUSSION TITLE
if you see the link in the description, somehow prism is not able to work here, other then here, it's working everywhere, please help.

### DISCUSSION THREAD

**Discussion by xnmn** [PrismJS/prism#3560] [Q&A]
http://seonewkhazana.ga/docs/getting-started/elements/update-department/#responses

**Comment by mAAdhaTTah** [comment]
Gonna guess this has something to do with it:

![Screen Shot 2022-09-20 at 10 11 53 AM](https://user-images.githubusercontent.com/4371429/191280924-760188ed-f120-41ce-b8a1-116e5b5c5b7c.png)

**Reply by xnmn** [reply]
hello, thank you so much for the response, any ideas on how to fix it? thanks

**Reply by xnmn** [reply]
as if you see here: http://seonewkhazana.ga/docs/getting-started/code/code-twilight-theme/ it works perfectly fine
